src/main/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefApiApplication.java

```diff
@@ -13,6 +15,7 @@
 @EnableJpaRepositories
 @SpringBootApplication
 @EnableCircuitBreaker
+@EnableCaching
 @EnableRetry
 @EnableFeignClients(basePackages = {
     "uk.gov.hmcts.reform.cwrdapi" },
```

Comment:
why do we have 
@EnableCircuitBreaker	and @EnableRetry here?

Reply:
I have no idea, but if we are not using them I can remove them

Reply:
yes pls!

Reply:
@EnableRetry I have added for retry mechanism of Publishing message to topic

Reply:
And for `@EnableCircuitBreaker` @SujeethKumarAri ?

Reply:
You can remove that :)

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/121/commits/df9ccbf9f01ea5953f7668c81e15f9a8e3d91f78